pkg/registry/apis/provisioning/resources/parser.go

```diff
@@ -241,7 +242,12 @@ func (f *ParsedResource) Run(ctx context.Context) error {
 		return fmt.Errorf("unable to find client")
 	}
 
-	var err error
+	// Always use the provisioning identity when writing
```

Comment:
currently we have other spots in which we also write / delete and we use the client differently (e.g. dual writer, resources/resource.go). 

Have you check if use a context with the provisioning Identity on writes?

Reply:
yes -- the other cases are all initiated from a context that is already service

This is the one case where a user initiates the request